INFO     graphviper:parameter.py:212 Module path: [38;2;50;50;205m/home/runner/work/testviper/testviper/src/toolviper[0m
WARNING  graphviper:logger.py:109 It is recommended that the local cache directory be set using the [38;2;50;50;205mdask_local_dir[0m parameter.
INFO     distributed.preloading:preloading.py:188 Creating preload: /home/runner/work/testviper/testviper/src/toolviper/src/toolviper/dask/plugins/scheduler.py
INFO     distributed.utils:utils.py:1145 Reload module scheduler from .py file
INFO     distributed.preloading:preloading.py:125 Import preload module: /home/runner/work/testviper/testviper/src/toolviper/src/toolviper/dask/plugins/scheduler.py
INFO     distributed.http.proxy:proxy.py:85 To route to workers diagnostics web server please install jupyter-server-proxy: python -m pip install jupyter-server-proxy
INFO     distributed.scheduler:scheduler.py:1766 State start
INFO     distributed.scheduler:scheduler.py:4282   Scheduler at:     tcp://127.0.0.1:43675
INFO     distributed.scheduler:scheduler.py:4297   dashboard at:  http://127.0.0.1:8787/status
INFO     distributed.preloading:preloading.py:200 Run preload setup: /home/runner/work/testviper/testviper/src/toolviper/src/toolviper/dask/plugins/scheduler.py
INFO     distributed.scheduler:scheduler.py:8182 Registering Worker plugin shuffle
INFO     distributed.nanny:nanny.py:368         Start Nanny at: 'tcp://127.0.0.1:34325'
INFO     distributed.nanny:nanny.py:368         Start Nanny at: 'tcp://127.0.0.1:32781'
INFO     distributed.scheduler:scheduler.py:4635 Register worker addr: tcp://127.0.0.1:46495 name: 1
INFO     distributed.scheduler:scheduler.py:6224 Starting worker compute stream, tcp://127.0.0.1:46495
INFO     distributed.core:core.py:883 Starting established connection to tcp://127.0.0.1:52716
INFO     distributed.scheduler:scheduler.py:4635 Register worker addr: tcp://127.0.0.1:37031 name: 0
INFO     distributed.scheduler:scheduler.py:6224 Starting worker compute stream, tcp://127.0.0.1:37031
INFO     distributed.core:core.py:883 Starting established connection to tcp://127.0.0.1:52730
INFO     distributed.scheduler:scheduler.py:5959 Receive client connection: MenrvaClient-5eedc304-890a-11f0-8de3-6045bd35d704
INFO     distributed.core:core.py:883 Starting established connection to tcp://127.0.0.1:52742
INFO     distributed.scheduler:scheduler.py:8182 Registering Worker plugin worker_logger
INFO     graphviper:logger.py:55 Client <MenrvaClient: 'tcp://127.0.0.1:43675' processes=2 threads=2, memory=5.59 GiB>
INFO     graphviper:parameter.py:212 Module path: [38;2;50;50;205m/home/runner/work/testviper/testviper/src/toolviper[0m
INFO     graphviper:logger.py:55 Downloading from [cloudflare] ....
INFO     graphviper:logger.py:55 Partition scheme that will be used: ['DATA_DESC_ID', 'OBS_MODE', 'OBSERVATION_ID', 'FIELD_ID']
INFO     graphviper:logger.py:55 Number of partitions: 12
INFO     graphviper:logger.py:55 OBSERVATION_ID [0], DDI [0], STATE [32 23 30 37], FIELD [0], SCAN [ 9 17 21 25]
INFO     graphviper:logger.py:55 OBSERVATION_ID [0], DDI [0], STATE [33 24 31], FIELD [1], SCAN [ 9 17 21]
INFO     graphviper:logger.py:55 OBSERVATION_ID [0], DDI [0], STATE [34 25 32], FIELD [2], SCAN [ 9 17 21]
INFO     graphviper:logger.py:55 OBSERVATION_ID [1], DDI [0], STATE [32 23 30 37], FIELD [0], SCAN [26 34 38 42]
INFO     graphviper:logger.py:55 OBSERVATION_ID [1], DDI [0], STATE [33 24 31], FIELD [1], SCAN [26 34 38]
INFO     graphviper:logger.py:55 OBSERVATION_ID [1], DDI [0], STATE [34 25 32], FIELD [2], SCAN [26 34 38]
INFO     graphviper:logger.py:55 OBSERVATION_ID [2], DDI [0], STATE [32], FIELD [0], SCAN [43]
INFO     graphviper:logger.py:55 OBSERVATION_ID [2], DDI [0], STATE [33], FIELD [1], SCAN [43]
INFO     graphviper:logger.py:55 OBSERVATION_ID [2], DDI [0], STATE [34], FIELD [2], SCAN [43]
INFO     graphviper:logger.py:55 OBSERVATION_ID [3], DDI [0], STATE [48 39 46 53], FIELD [0], SCAN [48 56 60 64]
INFO     graphviper:logger.py:55 OBSERVATION_ID [3], DDI [0], STATE [49 40 47], FIELD [1], SCAN [48 56 60]
INFO     graphviper:logger.py:55 OBSERVATION_ID [3], DDI [0], STATE [50 41 48], FIELD [2], SCAN [48 56 60]
ERROR    distributed.scheduler:scheduler.py:4840 'dependencies'
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/distributed/scheduler.py", line 4830, in _create_taskstate_from_graph
    plugin.update_graph(
    ~~~~~~~~~~~~~~~~~~~^
        self,
        ^^^^^
    ...<5 lines>...
        stimulus_id=stimulus_id,
        ^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/runner/work/testviper/testviper/src/toolviper/src/toolviper/dask/plugins/scheduler.py", line 87, in update_graph
    dependencies = kw["dependencies"]
                   ~~^^^^^^^^^^^^^^^^
KeyError: 'dependencies'